assets/scss/02-settings/_typography.scss

```diff
@@ -140,4 +142,11 @@ $typography-styles: (
             font-family: $typography-font-family-sans-serif,
         ),
     ),
+    tip-label: (
+        default: (
+            font-size: ms(-3),
+            line-height: 1.6,
```

Comment:
Nevyplatilo by se pro hodnoty `line-height` udělat mapu? Nebo je pro to nějaký důvod, proč tomu tak není, který mi teď nedochází? (Možná kvůli tomu, že výška řádku u `header-title` je definována pomocí mixinu pro odsazení?)

Reply:
Míříš správným směrem. Původní myšlenkou bylo držet tu přesný vertikální rytmus, a to právě s pomocí mixinu `spacing()`. Potíž je, že takhle jednoduše to nefunguje a brzy bylo potřeba upřednostňovat optický rytmus před exaktním, už jen se změnou velikosti písma ten systém přestává rychle fungovat, problematika je o dost složitější. Takže tu mám kromě `spacing()` nějakých 5–6 dalších hodnot. Zkusím dát do mapy aspoň ty hodnoty, které se opakují a jsou znovupoužitelné jinde, ale třeba u `pre` nejvíc sedí hodnota `1.5`, která jinde použitá není.